core/Session.php

```diff
@@ -185,4 +185,41 @@ public static function isSessionStarted()
     {
         return self::$sessionStarted;
     }
+
+    /**
+     * Write cookie header.  Similar to the native setcookie() function but also supports
+     * the SameSite cookie property.
+     * @param $name
+     * @param $value
+     * @param int $expires
+     * @param string $path
+     * @param string $domain
+     * @param bool $secure
+     * @param bool $httpOnly
+     * @param string $sameSite
+     * @return string
+     */
+    public static function writeCookie($name, $value, $expires = 0, $path = '/', $domain = '/', $secure = false, $httpOnly = false, $sameSite = 'lax')
+    {
+        $headerStr = 'Set-Cookie: ' . $name . '=' . $value;
```

Comment:
should the name and value by rawurlencoded for security etc? Ideally maybe also some other values?